app/src/main/java/org/jellyfin/androidtv/util/Utils.java

```diff
@@ -974,10 +975,11 @@ public static String GetProgramSubText(BaseItemDto baseItem) {
         Calendar start = Calendar.getInstance();
         start.setTime(Utils.convertToLocalDate(baseItem.getStartDate()));
         int day = start.get(Calendar.DAY_OF_YEAR);
-        return baseItem.getChannelName() + " - " + (baseItem.getEpisodeTitle() != null ? baseItem.getEpisodeTitle() : "") + " " +
-                ((day != Calendar.getInstance().get(Calendar.DAY_OF_YEAR) ? getFriendlyDate(start.getTime()) + " " : "") +
-                        android.text.format.DateFormat.getTimeFormat(TvApp.getApplication()).format(start.getTime()) + "-"
-                        + android.text.format.DateFormat.getTimeFormat(TvApp.getApplication()).format(Utils.convertToLocalDate(baseItem.getEndDate())));
+        return (baseItem.getChannelName() != null ? baseItem.getChannelName() + " - " : "") +
```

Comment:
Is there any way this can be broken up into logical parts? It's practically unreadable 😅

Reply:
Should be less "poetic" now. :joy: